.github/workflows/build_and_test_workflow_run.yml

```diff
@@ -0,0 +1,18 @@
+name: Cancelling Duplicates
+on:
+  workflow_run:
+    workflows: 
+      - 'Build and test'
+    types: ['requested']
+
+jobs:
+  cancel-duplicate-workflow-runs:
+    name: "Cancel duplicate workflow runs"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: potiuk/cancel-workflow-runs@953e057dc81d3458935a18d1184c386b0f6b5738
```

Comment:
If this is supposed to be maintained separately, we can even switch this to the proper release or using `master` branch since @potiuk seems a ASF committer / PMC in Airflow. However, using hash seems okay too for now.

Reply:
@potiuk recommends using commit hashes to your actions. See: https://github.com/apache/airflow/pull/13327/files
This way it is even in case of an attack on his Github account, the Apache infrastructure will be safe.

Reply:
👌